Added new ux writing chapters (#52)
siemens/ix-docs: docs/guidelines/language/numbers.mdx

---
sidebar_label: Numbers and percentages
title: Numbers and percentages
doc-type: banner
component-tabs:
  - ""
no_single_tab: true
description: In industrial settings, numbers and percentages are key to tracking performance, spotting trends, and making informed decisions. Since there are many ways to write them, the following guidelines aim to keep usage simple and consistent across teams.
---

import DoAndDont from '@site/src/components/DoAndDont';

# Numbers and percentages  

## General rules

> Write “Number” when there is enough space within the user interface.  

Reviewer: ```suggestion
Write “Number” or ”number” when there is enough space within the user interface.  
```

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>Number</DoAndDont.Item>
  </DoAndDont.Do>
  <DoAndDont.Dont>
    <DoAndDont.Item>Num.</DoAndDont.Item>
  </DoAndDont.Dont>
</DoAndDont>

Use “No.” as an abbreviation for number with a period and without a space.

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>No.6</DoAndDont.Item>
  </DoAndDont.Do>
  <DoAndDont.Dont>
    <DoAndDont.Item>Num.6</DoAndDont.Item>
  </DoAndDont.Dont>
</DoAndDont>

Avoid using the # symbol with numbers.  

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>Number 6</DoAndDont.Item>
  </DoAndDont.Do>
  <DoAndDont.Dont>
    <DoAndDont.Item>#6</DoAndDont.Item>
  </DoAndDont.Dont>
</DoAndDont>

> Write numbers as digits, such as “9” instead of “nine”, in industrial UIs.   

Reviewer: ```suggestion
Write numbers as digits, such as “9” instead of “nine”, in industrial user interfaces.   
```

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>We’ve released 3 new features this month.</DoAndDont.Item>
  </DoAndDont.Do>
  <DoAndDont.Dont>
    <DoAndDont.Item>We’ve released three new features this month.</DoAndDont.Item>
  </DoAndDont.Dont>
</DoAndDont>

> Write numbers with four digits and more with commas to separate thousands, millions, billions, etc. in American English.  

Reviewer: ```suggestion
Write numbers with four digits and more with commas to separate thousands, millions, etc. in American English.  
```

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>100,000</DoAndDont.Item>
  </DoAndDont.Do>
  <DoAndDont.Dont>
    <DoAndDont.Item>100.000</DoAndDont.Item>
  </DoAndDont.Dont>
</DoAndDont>

## Numbers and punctuation  

Use en dashes instead of hyphens for ranges. 

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>10–0</DoAndDont.Item>
     <DoAndDont.Item>The shipment weighs 50–75kg</DoAndDont.Item>
  </DoAndDont.Do>
</DoAndDont>

Write out the unit only once in ranges. 

<DoAndDont>
  <DoAndDont.Do>
    <DoAndDont.Item>The temperature fluctuated between 20–25°C</DoAndDont.Item>
     <DoAndDont.Item>The temperature fluctuated between 20°C–25°C</DoAndDont.Item>
  </DoAndDont.Do>
</DoAndDont>

Use the minus hyphen for negative numbers which is better aligned than a normal hyphen (Unicode U+2212).  

<DoAndDont>
	<DoAndDont.Do>
		<DoAndDont.Item>-12</DoAndDont.Item>
	</DoAndDont.Do>
	<DoAndDont.Dont>
		<DoAndDont.Item>—12</DoAndDont.Item>
	</DoAndDont.Dont>
</DoAndDont>

Use either “to” or separate text fields for ranges with minus numbers for readability.  

<DoAndDont>
	<DoAndDont.Do>
		<DoAndDont.Item>-12 to -15°C</DoAndDont.Item>

> 		<DoAndDont.Item>Min temp: -12 / Max temp: -15°</DoAndDont.Item>

Reviewer: ```suggestion
		<DoAndDont.Item>Min temp (C): -12° / Max temp (C): -15°</DoAndDont.Item>
```

		<DoAndDont.Item>Our goal is to deliver software to millions of happy users worldwide</DoAndDont.Item>
	</DoAndDont.Do>
</DoAndDont>

Write out numbers in the millions and billions (“one million” instead of “1,000,000”) in longer texts and paragraphs. 

<DoAndDont>
	<DoAndDont.Do>
		<DoAndDont.Item>We have over one million users worldwide.</DoAndDont.Item>
	</DoAndDont.Do>
	<DoAndDont.Dont>
		<DoAndDont.Item>We have over 1,000,000 users worldwide.</DoAndDont.Item>
	</DoAndDont.Dont>
</DoAndDont>

Use numerals for numbers in the millions for dates, addresses, units of measurement, currency and percentages. 

<DoAndDont>
	<DoAndDont.Do>
		<DoAndDont.Item>The company headquarters is located at 1,000,000 Main Street.</DoAndDont.Item>
		<DoAndDont.Item>The temperature can reach 1,000,000°C in the pod.</DoAndDont.Item>
		<DoAndDont.Item>The cargo ship can carry up to 1,000,000kg.</DoAndDont.Item>
		<DoAndDont.Item>The generator’s top speed is 1,000,000km/h.</DoAndDont.Item>
		<DoAndDont.Item>The success rate of our new implementation is 1,000,000%.</DoAndDont.Item>
		<DoAndDont.Item>The project budget is $1,000,000. </DoAndDont.Item>
	</DoAndDont.Do>
</DoAndDont>

## Percentages 

> Use % instead of “percentage” within the user interface to save space. 

Reviewer: ```suggestion
Use the % symbol instead of writing out “percentage” within the user interface to save space. 
```

<DoAndDont>
	<DoAndDont.Do>
		<DoAndDont.Item>10%</DoAndDont.Item>
	</DoAndDont.Do>
	<DoAndDont.Dont>
		<DoAndDont.Item>10 percent</DoAndDont.Item>
	</DoAndDont.Dont>
</DoAndDont>

Use % after the number and without a space.  

<DoAndDont>
	<DoAndDont.Do>
		<DoAndDont.Item>10%</DoAndDont.Item>
	</DoAndDont.Do>
	<DoAndDont.Dont>
		<DoAndDont.Item>%10</DoAndDont.Item>
	</DoAndDont.Dont>
</DoAndDont>

## Phone numbers 

> Separate phone numbers into groups for clarity. 

Reviewer: ```suggestion
Separate phone numbers into groups for clarity when backend localization or separate fields are not possible. 
```
